spec/services/admin/flags/remove_submission_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe Admin::Flags::RemoveSubmission do
+  let(:admin) { create(:user, admin: true) }
```

Comment:
Just a nitpick, can we have the subject be up top with the two lets below please? 

```suggestion
  subject(:service) { described_class.call(admin, flag: flag) }
  
  let(:admin) { create(:user, admin: true) }
  let(:flag) { create(:flag) }
```

Reply:
Yep definitely looks better that way. I noticed that `Dismiss` and `BanUser` also did not have `subject` at the top, so most recent commit also changes those. Hope that's okay.